components/brave_rewards/resources/ui/stories/modal.tsx

```diff
@@ -332,7 +332,7 @@ storiesOf('Rewards/Modal', module)
     return (
       <ModalRedirect
         titleText={text('Title text', 'Sorry there was problem processing your request, please try again.')}
-        errorText={{ __html: 'Error explanation, more info <a href="#">here</a>.' }}
```

Comment:
This just says 'error explanation' so i'm not sure if it's ever displayed in production?

Reply:
This is a storybook module, so it's not a part of the browser build.